packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -276,6 +276,14 @@ interface DatabaseClusterBaseProps {
    */
   readonly storageEncryptionKey?: kms.IKey;
 
+  /**
+   * The storage type to be associated with the DB cluster.
+   * Valid values: `aurora` , `aurora-iopt1` (Aurora DB clusters); `io1` (Multi-AZ DB clusters)
+   *
+   * @default: `aurora` (Aurora DB clusters); `io1` (Multi-AZ DB clusters)
+   */
+  readonly storageType?: string;
```

Comment:
Can you change this into an enum?

Reply:
@corymhall Could you help review the updated version? I have changed this to enum.